doc/features.mld

```diff
@@ -411,7 +411,7 @@ module type S =
   end
 ]}
 
-whereas [odoc] preserves the fact that [Y] and [Z] are calculated from [X]. If the 
+Whereas [odoc] preserves the fact that [Y] and [Z] are calculated from [X]. If the 
```

Comment:
It's not mandatory. It's best in technical docs to have shorter sentences when possible, and sentences that have a code block in the middle can be confusing. 

How about: "On the other hand, `odoc` preserves the fact.... "
As you're saying OCaml does X, but `odoc` does Y. Correct?

If you don't like that, we can go back to the original. You're right that it's the same sentence.

IRT li 427, I think it should be a new sentence starting with "The signature..."
How would this be:

"....the results would be different. For example: 
`code block`
 The signature of `T`, as calculated by OCaml, will be 
`code block`"

Although in a previous note you said that "For example:" doesn't work because it's not the entire example, so perhaps we keep it as it is, but it still needs to start with "the signature....will then be" -- "then" is in a weird place along with the code block.

Reply:
For line 427, your suggestion:

> "....the results would be different. For example:
`code block`
The signature of T, as calculated by OCaml, will be
`code block`"

looks good I think!

Yes, I agree that sentences with code blocks in the middle is confusing and should be avoided! For some reason, "Whereas, ..." does not a complete sentence, _whereas_ "On the other hand, ..." does look to be a complete sentence!
But I'll leave you the judge on this one!